src/core/model_config.proto

```diff
@@ -713,6 +713,85 @@ message ModelQueuePolicy
 //@@
 message ModelDynamicBatching
 {
+  //@@  .. cpp:var:: message BatchInput
+  //@@
+  //@@     A batch input is an additional input for the requests in the batch
```

Comment:
"A batch input is an additional input that must be added by the backend based on all the requests in a batch."

Let's not tie this specifically to ragged input as we may find other uses for it in the future.